data/json/vitrification/furniture_vitrified.json

```
@@ -0,0 +1,880 @@
+    "symbol": "@",
+    "color": "dark_gray",
+    "looks_like": "f_statue",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_9",
+    "name": "black glass figure",
+    "description": "A black glass figure of someone doubled over in laughter.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "looks_like": "f_statue",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_10",
+    "name": "black glass figure",
+    "description": "A black glass figure. Their features are too jagged to make out clearly, but they remind you of someone you knew.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "looks_like": "f_statue",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_11",
+    "name": "black glass figure",
+    "description": "A black glass figure of someone in a fighting stance, their face twisted with howls of rage and anger.  They're gripping some sort of weapon that isn't there.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "looks_like": "f_statue",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_bed",
+    "name": "black glass bed",
+    "symbol": "#",
+    "description": "A black glass sculpture of a bed. The rumpled covers are jagged, and splinter at the touch.",
+    "color": "dark_gray",
+    "move_cost_mod": 3,
+    "coverage": 40,
+    "comfort": 0,
+    "floor_bedding_warmth": 0,
+    "required_str": -1,
+    "deconstruct": { "items": [ { "item": "black_glass_shard", "count": [ 10, 20 ] } ] },
+    "flags": [ "TRANSPARENT", "PLACE_ITEM", "MOUNTABLE", "CAN_SIT", "EASY_DECONSTRUCT" ],
```

```suggestion
    "flags": [ "TRANSPARENT", "PLACE_ITEM", "MOUNTABLE", "CAN_SIT", "EASY_DECONSTRUCT" ],
```
I haven't checked flags much but EASY_DECONSTRUCT seems out of place